test/README.md

```diff
@@ -0,0 +1,63 @@
+[![Build Status](https://travis-ci.org/webrtc/samples.svg)](https://travis-ci.org/webrtc/samples)
+
+# Intro #
+Selenium WebDriver, Node, Testling and travis-multirunner are used as the testing framework. Selenium WebDriver drives the browser; Node and Testling manage the tests, while travis-multirunner downloads and installs the browsers to be tested on, i.e. creates the testing matrix.
+
+## Development ##
+Detailed information on developing in the [webrtc](https://github.com/webrtc) GitHub repo can be found in the [WebRTC GitHub repo developer's guide](https://docs.google.com/document/d/1tn1t6LW2ffzGuYTK3366w1fhTkkzsSvHsBnOHoDfRzY/edit?pli=1#heading=h.e3366rrgmkdk).
+
+This guide assumes you are running a Debian based Linux distribution (travis-multirunner currently fetches .deb browser packages).
+
+#### Clone the repo in desired folder
+```bash
+git clone https://github.com/webrtc/samples.git
+```
+
+#### Install npm dependencies
+```bash
+sudo npm install
```

Comment:
Is sudo required here? Aren't these installed "local" rather than "global" (-g) in npm parlance? - I'm configured with a ~/npm-global prefix, so I can't tell. Can others check?